src/main/java/org/tb/dailyreport/rest/DailyReportRestEndpoint.java

```diff
@@ -121,11 +120,9 @@ public void createBooking(@RequestBody DailyReportData booking) {
                 1
             );
         } catch (AuthorizationException e) {
-            throw new ResponseStatusException(UNAUTHORIZED, "Could create timereport. " + e.getErrorCode());
+            throw new ResponseStatusException(UNAUTHORIZED, "Could not create timereport. " + e.getErrorCode());
```

Comment:
Besser ErrorCodeException catchen, ansonsten werden wichtige Exception nicht in eine ResponseStatusException übersetzt

Reply:
catchen der exception macht keinen sinn - da kein service aufgerufen wird und das dao so gut wie keine logik enthält